pkg/policies/branch/branch.go

```diff
@@ -321,6 +330,14 @@ func check(ctx context.Context, rep repositories, c *github.Client, owner,
 				d.BlockForce = false
 			}
 		}
+		ea := p.GetEnforceAdmins()
+		d.EnforceOnAdmins = false
```

Comment:
```
d.EnforceOnAdmins = (ea != nil  && ea.Enabled)
```

Reply:
Fixed in 2cbc17cb002f77b3260b407bc8ca273fc14abb0d